From d6236220e351d7b712825997b7d84feb38e49bd3 Mon Sep 17 00:00:00 2001
From: Colin Watson <cjwatson@debian.org>
Date: Mon, 23 Mar 2015 14:34:01 +0000
Subject: hostfs: Drop unnecessary feature test macros

_BSD_SOURCE was added to allow the use of DT_DIR, but that was removed
in e768b77068a0b030a07576852bd0f121c9a077eb.  While adding
_DEFAULT_SOURCE as well works around problems with current glibc,
neither is in fact needed nowadays.

Origin: upstream, http://git.savannah.gnu.org/cgit/grub.git/commit/?id=a276b840464629fdaecbe7e813f83a717a34993a
Origin: upstream, http://git.savannah.gnu.org/cgit/grub.git/commit/?id=5974d4ba65fc5ad9253857bf992ffd3c635baf2a
Last-Update: 2015-03-23

Patch-Name: hostfs_remove_ftm.patch
---
 grub-core/kern/emu/hostfs.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/grub-core/kern/emu/hostfs.c b/grub-core/kern/emu/hostfs.c
index 7e725f6..7b28c00 100644
--- a/grub-core/kern/emu/hostfs.c
+++ b/grub-core/kern/emu/hostfs.c
@@ -19,7 +19,6 @@
 
 #include <config-util.h>
 
-#define _BSD_SOURCE
 #include <grub/fs.h>
 #include <grub/file.h>
 #include <grub/disk.h>
